### PR TITLE
[WIP] adding moa quickstart and initial structure

### PR DESCRIPTION
**Pull request by lamek**
This PR contains the initial structure for the moa doc set, as well as the getting started guide.

### CODE REVIEW

**Comment by lamek** [comment]
Preview here - http://file.bos.redhat.com/klamenzo/moa/4/getting_started_moa/creating-first-moa-cluster.html
(note you need to be on the VPN to access this)

File: getting_started_moa/creating-first-moa-cluster.adoc [hunk 1]
```diff
@@ -0,0 +1,46 @@
+:context: creating-first-moa-cluster
+[id="creating-first-moa-cluster"]
+= Getting started with {product-title}
+include::modules/common-attributes.adoc[]
+
+toc::[]
+
+This guide walks through setting up your first {product-title} cluster using `moactl`.
```
Comment (sfortner-RH) [inline getting_started_moa/creating-first-moa-cluster.adoc:8]: Do we want to refer to this as a Tutorial vs Guide? To me a guide encompasses all of the product documentation, but in a Getting Started format it's more of  Tutorial structure. Just a thought. I'm fine with Guide if that's consistent with other RH docs.

File: getting_started_moa/creating-first-moa-cluster.adoc [hunk 2]
```diff
@@ -0,0 +1,46 @@
...
+
```
Comment (sfortner-RH) [inline getting_started_moa/creating-first-moa-cluster.adoc:9]: I like this TL;DR version for a user that doesn't want to walk through the full tutorial and feels confident using the below commands.

File: getting_started_moa/creating-first-moa-cluster.adoc [hunk 3]
```diff
@@ -0,0 +1,46 @@
...
+## Promotes a user from your IDP to admin level
+$ moactl create user --cluster <my-cluster-name> --dedicated-admins <admin-username>        
+
+## Checks if your install is ready (look for State: Ready),
+## and provides your Console URL to login to the web console.
+$ moactl describe cluster <my-cluster-name>
+----
+
+If you are just starting out, or if you get stuck, consult the rest of this guide for more information on each step.
+
```
Comment (sfortner-RH) [inline getting_started_moa/creating-first-moa-cluster.adoc:35]: I would combine these two sentences. 

"For additional help, consult the rest of this tutorial for guided steps on running an Amazon Red Hat OpenShift cluster on your AWS account. " ?

File: modules/moa-installing-prerequisites.adoc [hunk 4]
```diff
@@ -0,0 +1,216 @@
...
+
+
+[id="moa-installing-prerequisites"]
+= Installing prerequisites
+
+Complete the following prerequisites before creating your {product-title} cluster.
+
+.Procedure
+
+. Select an AWS account to use
```
Comment (sfortner-RH) [inline modules/moa-installing-prerequisites.adoc:13]: Should each step be a complete sentence? "Select the AWS account you want to use."

File: modules/moa-installing-prerequisites.adoc [hunk 5]
```diff
@@ -0,0 +1,216 @@
...
++
+Unless your just testing out MOA, we recommend using a dedicated AWS account to run any production clusters. If you are utilizing AWS Organizations, you can use an AWS account within your organization or link:https://docs.aws.amazon.com/organizations/latest/userguide/orgs_manage_accounts_create.html#orgs_manage_accounts_create-new[create a new one].
```
Comment (sfortner-RH) [inline modules/moa-installing-prerequisites.adoc:15]: Unless you're*

File: modules/moa-installing-prerequisites.adoc [hunk 6]
```diff
@@ -0,0 +1,216 @@
...
++
+If you are using AWS organizations and you need to have a Service Control Policy (SCP) applied to the AWS account you plan to use, see the link:https://www.openshift.com/dedicated/ccs#scp[Red Hat Requirements for Customer Cloud Subscriptions] for details on the minimum required SCP.
++
+As part of the cluster creation process, `moactl` will perform the following actions:
++
+* Create an osdCcsAdmin IAM user:
+** This user will have Programmatic access enabled.
+** This user will have the AdministratorAccess policy attached to it.
+** This user will leverage the IAM credentials you provide when configuring the AWS cli in the next section.
+. Install and configure the AWS cli
```
Comment (sfortner-RH) [inline modules/moa-installing-prerequisites.adoc:25]: Add period to the end [complete sentence]

File: modules/moa-installing-prerequisites.adoc [hunk 7]
```diff
@@ -0,0 +1,216 @@
...
+.. Install the link:https://aws.amazon.com/cli/[aws-cli].
+.. Configure the `aws-cli` to use the AWS account that you would like to deploy your cluster into. .... Modify your `~/.aws/credentials` file to specify the correct `aws_access_key_id` and `aws_secret_access_key`.
```
Comment (sfortner-RH) [inline modules/moa-installing-prerequisites.adoc:27]: Remove "...."

File: modules/moa-installing-prerequisites.adoc [hunk 8]
```diff
@@ -0,0 +1,216 @@
...
+
+.Procedure
+
+. Select an AWS account to use
++
+Unless your just testing out MOA, we recommend using a dedicated AWS account to run any production clusters. If you are utilizing AWS Organizations, you can use an AWS account within your organization or link:https://docs.aws.amazon.com/organizations/latest/userguide/orgs_manage_accounts_create.html#orgs_manage_accounts_create-new[create a new one].
++
+If you are using AWS organizations and you need to have a Service Control Policy (SCP) applied to the AWS account you plan to use, see the link:https://www.openshift.com/dedicated/ccs#scp[Red Hat Requirements for Customer Cloud Subscriptions] for details on the minimum required SCP.
++
+As part of the cluster creation process, `moactl` will perform the following actions:
```
Comment (sfortner-RH) [inline modules/moa-installing-prerequisites.adoc:19]: The way it's currently structured there is only one action listed, creating the osdcCcsAdmin IAM user.  And once this is done, by default the user inherits specific permissions and policies? I would remove the bulleted list here, unless you plan to add more actions.

"As part of the cluster creation process, moactl will create an osdCcsAdmin IAM user. This user will leverage the IAM credentials you provide when configuring the AWS cli in the next section.
Note: This user will have Programmatic access enabled and the AdministratorAccess policy attached to it."

File: modules/moa-installing-prerequisites.adoc [hunk 9]
```diff
@@ -0,0 +1,216 @@
...
+||  ec2.ap-southeast-2.amazonaws.com |  opt-in-not-required  |  ap-southeast-2 ||
+||  ec2.eu-central-1.amazonaws.com   |  opt-in-not-required  |  eu-central-1   ||
+||  ec2.us-east-1.amazonaws.com      |  opt-in-not-required  |  us-east-1      ||
+||  ec2.us-east-2.amazonaws.com      |  opt-in-not-required  |  us-east-2      ||
+||  ec2.us-west-1.amazonaws.com      |  opt-in-not-required  |  us-west-1      ||
+||  ec2.us-west-2.amazonaws.com      |  opt-in-not-required  |  us-west-2      ||
+|+-----------------------------------+-----------------------+-----------------+|
+----
++
+. Install `moactl`, the {product-title} command line utility.
```
Comment (sfortner-RH) [inline modules/moa-installing-prerequisites.adoc:78]: Should we explain what moactl is at the beginning of this getting started guide? I got to this step and was like "Oh, that's what maoctl is!", and then I was like wait did we talk about it already. It's mentioned twice above, but never outlined until this step. So the user doesn't know what moactl is until step 3 despite the previous mentions.

File: modules/moa-installing-prerequisites.adoc [hunk 10]
```diff
@@ -0,0 +1,216 @@
...
+. Run the following command to verify that your AWS account has the necessary permissions:
++
+[source,bash]
+----
+$ moactl verify permissions
+I: Validating SCP policies...
+I: AWS SCP policies ok
+----
++
+.. Verify that your AWS account has the necessary quota to deploy an OpenShift cluster. Sometimes quota varies by region, which may prevent you from deploying:
```
Comment (sfortner-RH) [inline modules/moa-installing-prerequisites.adoc:130]: The note regurgitates the 2nd sentence. So I would remove it here.

File: modules/moa-installing-prerequisites.adoc [hunk 11]
```diff
@@ -0,0 +1,216 @@
...
+[NOTE]
+====
+Sometimes quota varies by region, which may prevent you from deploying. If you receive any errors, try a different region.
+====
++
+If you need to increase your quota, navigate to your link:https://aws.amazon.com/console/[AWS console], and request a quota increase for the service that failed.
++
+Once both the permissions and quota checks pass, proceed to initializing your AWS account.
++
+. Initializing your AWS account
```
Comment (sfortner-RH) [inline modules/moa-installing-prerequisites.adoc:149]: 5. initialize your AWS account using moactl.

updating the step, you could remove the lead in sentence before the substeps

File: modules/moa-installing-prerequisites.adoc [hunk 12]
```diff
@@ -0,0 +1,216 @@
...
+OCM Account ID:               1DzGIdIhqEWyt8UUXQhSoWaaaaa
+OCM Account Name:             Your Name
+OCM Account Username:         you@domain.com
+OCM Account Email:            you@domain.com
+OCM Organization ID:          1HopHfA2hcmhup5gCr2uH5aaaaa
+OCM Organization Name:        Red Hat
+OCM Organization External ID: 0000000
+----
++
+. Initialize your AWS account
```
Comment (sfortner-RH) [inline modules/moa-installing-prerequisites.adoc:190]: Step 5 and 7 are both initializing the AWS account. Are there two ways to do it? Or can they be combined?

**Comment by lamek** [comment]
FYI we're going to merge these changes directly to enterprise-4.3.
Distro map changes to master are in a separate PR - https://github.com/openshift/openshift-docs/pull/23896

File: modules/moa-installing-prerequisites.adoc (hunk 8 above)
Comment (lamek) [reply modules/moa-installing-prerequisites.adoc:19]: Great catch. Yes, we used to have more actions but decided spelling them all out here was not a good choice.

File: modules/moa-installing-prerequisites.adoc (hunk 12 above)
Comment (lamek) [reply modules/moa-installing-prerequisites.adoc:190]: I updated 5 to be "Prepare your AWS account for cluster deployment." I made a number of other small changes: the steps in this section should all be under this heading, so I updated the nesting.

File: modules/moa-installing-prerequisites.adoc (hunk 9 above)
Comment (lamek) [reply modules/moa-installing-prerequisites.adoc:78]: Good call. I added this line to the end of the very first sentence in the module: "...the {product-title} command line utility."

**Comment by lamek** [comment]
Moving this to a new PR.
https://github.com/openshift/openshift-docs/pull/23900

**Pull request closed without merging**